src/acceptance-test/java/org/zalando/nakadi/webservice/TimelinesControllerAT.java

```diff
@@ -6,28 +6,36 @@
 import com.jayway.restassured.http.ContentType;
 import org.apache.http.HttpStatus;
 import org.hamcrest.Matchers;
+import org.joda.time.DateTime;
 import org.json.JSONObject;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
 import org.junit.Test;
 import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.domain.EventTypeOptions;
 import org.zalando.nakadi.utils.EventTypeTestBuilder;
 import org.zalando.nakadi.webservice.utils.NakadiTestUtils;
 
+import java.time.Duration;
+
+import static java.time.temporal.ChronoUnit.DAYS;
+import static java.time.temporal.ChronoUnit.MINUTES;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.zalando.nakadi.util.DateWithinMatcher.dateWithin;
+
 public class TimelinesControllerAT extends BaseAT {
 
-    private static EventType eventType = EventTypeTestBuilder.builder().build();
+    private static final long RETENTION_TIME_MS = Duration.of(2, DAYS).toMillis();
 
-    @BeforeClass
-    public static void setUp() throws JsonProcessingException {
-        NakadiTestUtils.createEventTypeInNakadi(eventType);
-    }
+    private final EventType eventType;
 
-    @AfterClass
-    public static void tearDown() {
```

Comment:
I don't really see a reason to cleanup. In out ATs we usually don't cleanup because it will just add extra time to execution and it is not really needed as these tests are never run on real DBs. 
The only exception is UserJourneyAT, we run it on real environments and there we do cleanup after test.